newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -116,20 +113,47 @@ export const GameDetailsDialog = ({
     [loadGameMetrics]
   );
 
+  const loadPublicGame = React.useCallback(
+    async () => {
+      setPublicGameError(null);
+      try {
+        const publicGameResponse = await getPublicGame(game.id);
+        setPublicGame(publicGameResponse);
+      } catch (err) {
+        console.error(`Unable to load the game:`, err);
+        setPublicGameError(err);
+      }
+    },
+    [game]
+  );
+
+  React.useEffect(
+    () => {
+      loadPublicGame();
+    },
+    [loadPublicGame]
+  );
+
   const updateGameFromProject = async () => {
     if (!project || !profile) return;
     const { id } = profile;
 
     try {
-      const updatedGame = await updateGame(
-        getAuthorizationHeader,
-        id,
-        project.getProjectUuid(),
-        {
-          authorName: project.getAuthor(),
-          gameName: project.getName(),
-        }
-      );
+      setPublicGame(null); // Public game will auto update when game is updated.
+      const gameId = project.getProjectUuid();
+      const updatedGame = await updateGame(getAuthorizationHeader, id, gameId, {
+        authorName: project.getAuthor() || 'Unspecified publisher',
+        gameName: project.getName() || 'Untitle game',
+      });
+      const authorIds = project.getAuthorIds();
+      if (authorIds) {
+        const authorAcls = authorIds.toJSArray().map(authorId => ({
```

Comment:
Nit: this construction appears twice, let's put it in a function "authorIdsToGameUserAcls"